src/components/LibraryDetailPage.tsx

```diff
@@ -92,11 +96,16 @@ export class LibraryDetailPage extends React.Component<LibraryDetailPageProps, L
 
     const categories = {
       "Basic Information": "basic_info",
-      "Contact & URLs": "urls_and_contact"
+      "Contact & URLs": "urls_and_contact",
+      "Areas": "areas"
     };
-
-    Object.keys(categories).forEach(tabName => {
-      tabItems[tabName] = this.renderItems(library[categories[tabName]]);
+    Object.entries(categories).filter(([k, v]) => library[v]).forEach(([k, v]) => {
```

Comment:
What are you filtering here for?

Reply:
I had been filtering to make sure that the library object actually has this category (under the assumption that `library[areas]`--and probably other things as we add more features!--might not exist).  But, looking back at the latest version of the server code, I realize that there's no way for that to happen (if the library doesn't have areas, then `library[areas]` == `{focus: [], service: []}`); will remove the filter!